it/people/fortunata-grande.md

```diff
@@ -11,7 +11,7 @@ linkedin_url:
 ref: fortunata-grande
 parent_ref: team
 start_date: 1 febbraio 2018
-period_provided: fino al 15 settembre 2018
+period_provided: fino al 15 settembre 2019
```

Comment:
va creata pagina corrispettiva in inglese

Reply:
Confermo, non manca solo Fortunata, ma tutti i 7 profili degli amministrativi in inglese

Reply:
Mancano, ma mancavano anche prima di questa PR e quindi non possiamo bloccarla per questo motivo. Vanno fatte le traduzioni e poi aperta un'altra PR